README.md

```diff
@@ -1,26 +1,34 @@
 # How it works
-Press and hold letter then hit space bar or arrow key to select the accent. For example `e` & `⎵`. With spacebar, repeat stroke to change letter. When releasing the letter key, the accent is inserted.
 
-The software is currently working with the most of accents for several countries, including `Czech`, `German`, `France`, `Maori`, and many others... (the list can be found [here](https://github.com/damienleroy/PowerAccent/blob/main/PowerAccent.Core/Languages.cs)). If any langage is missing, don't hesitate to create an [issue](https://github.com/damienleroy/PowerAccent/issues).
+Press and hold a letter that supports diacritic marks, then press space bar or arrow key to select the accent. For example <kbd>E</kbd> + <kbd>Space</kbd>. With spacebar, repeat presses to change accent mark. When releasing the letter key, the accented letter is inserted.
 
-All accents are ordered by usage frequency ([Wikipedia](https://en.wikipedia.org/wiki/Letter_frequency)) by default.
+The software is currently working with the most of accents for several (Latin script based) countries, including **Czech**, **German**, **France**, **Maori** and many others. The list can be found in [Languages.cs](https://github.com/damienleroy/PowerAccent/blob/main/PowerAccent.Core/Languages.cs). If any language is missing, don't hesitate to create an [issue](https://github.com/damienleroy/PowerAccent/issues).
+
+All letters are sorted by usage frequency by default, as found on [Wikipedia](https://en.wikipedia.org/wiki/Letter_frequency).
 
 # Download
-See the [Release](https://github.com/leroyd/PowerAccent/releases) page
 
-# Problem known
-- Some keys can have weird behavior with some actions or some tools
-- (Experimental feature) Inside browsers & some other softwares, the tool can't detect the caret position. So default is applied, meaning up center of the main window
+See the [Release](https://github.com/leroyd/PowerAccent/releases) page.
```

Comment:
Are you using two repositories for the same project?

Reply:
haha, no. I changed the name of the repo since started the project.

Reply:
Sorry, I didn't mean the _repo_ name, I meant the `leroyd` part, but I think it auto-forwards to the correct account. Still better to use current name in docs.

Reply:
Yeah sorry, I would say the account name. Changed it from `leroyd` to `damienleroy`.